rero/rero-ils
# US696: overdue fees

* Create and adapt fees resource (module).
    * Create tests.
    * Create fixtures.
    * Update circ_pol JSON Schema and editor.
    * Create fees when generating notification.
    * Add default currency in config.py
    * add default currency at the organisation level
    * Closes #363
    * Structure editor with section.
    * Add overdue amount and currency in circ policies editor.
    * Add an allow renewals button calculated dynamically.

Co-Authored-by: Laurent Dubois <laurent.dubois@itld-solutions.be>
Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>
Co-Authored-by: Benoit Erken <benoit.erken@uclouvain.be>
Co-Authored-by: Aly Badr <aly.badr@rero.ch>


How to test:

* login as librarian or system librarian
* edit any circulation policy and validate the
	 checkout and renewal fields and
         the new fees field
* An overdue item is available at (https://ilsdev.test.rero.ch/global/items/3)
* Make sure a fine is already created at (https://ilsdev.test.rero.ch/api/fees/1)
